requirements_dev.txt

```diff
@@ -9,7 +9,7 @@ jupyter_contrib_nbextensions
 pep8-naming
 pre-commit==2.2.0
 pylint==2.4.4
-pytest
+pytest==5.4.1
 pytest-cov
 scipy
-tox
+tox==3.14.6
```

Comment:
why only partly specified?

Reply:
`pep8-naming` etc are `flake8` extensions and follow (afaik) the version of `flake8`. Similar for `pytest-cov`. `scipy` is only there because it's used in some tests, I'm not sure what to do with that one.

Reply:
Ok. You can leave scipy for now. We should try to get rid of it then later on. Its was used to do some comparison of data and models based on chi2, but I think the usage of scipy is so minimal that we can try to avoid that dependency.